volatility/framework/plugins/windows/netscan.py

```diff
@@ -0,0 +1,306 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+import datetime
+from typing import Iterable, List
+
+from volatility.framework import constants, exceptions, interfaces, renderers, symbols, layers
+from volatility.framework.automagic.pdbscan import scan
```

Comment:
To avoid namespace pollution we try not to import method, but only import modules.  So this would end up being `from volatility.framework.automagic import pdbscan` and then later on `scan` would become `pdbscan.scan`.

Reply:
that import is unused anyway and was left over from a previous attempt to determine driver version. I removed it ;)